README.md

```diff
@@ -20,33 +23,41 @@ Name                                      Stmts   Miss  Cover   Missing
 learning_journal/models/__init__.py          22      0   100%
 learning_journal/models/entries.py            9      0   100%
 learning_journal/models/meta.py               5      0   100%
+learning_journal/security.py                 30      0   100%
 learning_journal/views/data/__init__.py       0      0   100%
 learning_journal/views/data/entries.py        1      0   100%
-learning_journal/views/default.py            39      5    87%   80-84
+learning_journal/views/default.py            54      0   100%
 learning_journal/views/notfound.py            4      0   100%
 -----------------------------------------------------------------------
-TOTAL                                        80      5    94%
+TOTAL                                       125      0   100%
+
+46 passed in 11.41 seconds
+
 
-======================================================
+py36 runtests: PYTHONHASHSEED='1440921674'
+py36 runtests: commands[0] | py.test --cov=learning_journal -q --cov-report term-missing
+..............................................
 
 ---------- coverage: platform darwin, python 3.6.1-final-0 -----------
 Name                                      Stmts   Miss  Cover   Missing
 -----------------------------------------------------------------------
 learning_journal/models/__init__.py          22      0   100%
 learning_journal/models/entries.py            9      0   100%
 learning_journal/models/meta.py               5      0   100%
+learning_journal/security.py                 30      0   100%
 learning_journal/views/data/__init__.py       0      0   100%
 learning_journal/views/data/entries.py        1      0   100%
-learning_journal/views/default.py            39      5    87%   80-84
+learning_journal/views/default.py            54      0   100%
 learning_journal/views/notfound.py            4      0   100%
 -----------------------------------------------------------------------
-TOTAL                                        80      5    94%
+TOTAL                                       125      0   100%
+
+46 passed in 10.39 seconds
+
 
-27 passed in 2.12 seconds
-__________________________________________________________ 
-__________________________________________________________
   py27: commands succeeded
   py36: commands succeeded
+  congratulations :)
 
 
```

Comment:
You should consider adding a more step by step process for how to get a user up and running with this. Including commands and any environment variables that have to be set.